app/styles/brands/scielo.scss

```diff
@@ -8,7 +8,7 @@
     white,
     $logo-dir + 'scielo-banner.png',
     $logo-dir + 'scielo-logo.png',
-    true,
+    false,
```

Comment:
Yeah, this is not going to be a long term maintenance problem, all these unlabelled descriptors. :) I look forward to putting this into a database.